### PR TITLE
Fix locales to en_US.UTF-8

### PR DESCRIPTION
**Pull request by gangliao**
```bash
 File "/usr/lib/python2.7/locale.py", line 581, in setlocale
[12:22:32][Step 1/1]     return _setlocale(category, locale)
[12:22:32][Step 1/1] locale.Error: unsupported locale setting
```

### CODE REVIEW

File: paddle/scripts/docker/build.sh [hunk 1]
```diff
@@ -146,13 +144,18 @@ cat >> /paddle/build/Dockerfile <<EOF
...
+    apt-get install -y python-pip locales && \
+    pip install -U pip && \
     dpkg -i /*.deb ; apt-get install -f -y && \
     apt-get clean -y && \
-    rm -f /*.deb && \
-    paddle version
+    rm -f /*.deb
+
+# Fix locales to en_US.UTF-8
+RUN localedef -i en_US -f UTF-8 en_US.UTF-8
```
Comment (wangkuiyi) [inline paddle/scripts/docker/build.sh]: I read the manpage of [localedef](https://linux.die.net/man/1/localedef). But I don't get it why we need to call it here?

Comment (gangliao) [inline paddle/scripts/docker/build.sh]: I remember there will appear some messy code in Docker if we do not set en_US.UTF-8. Maybe I am wrong and I will verify this in Docker today.

Comment (typhoonzero) [inline paddle/scripts/docker/build.sh]: Found some workaround at: https://askubuntu.com/questions/581458/how-to-configure-locales-to-unicode-in-a-docker-ubuntu-14-04-container 

Try: 
```
RUN locale-gen en_US.UTF-8
ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
```
?

Comment (wangkuiyi) [inline paddle/scripts/docker/build.sh]: @typhoonzero What is the purpose of these commands?

Comment (gangliao) [inline paddle/scripts/docker/build.sh]: Just a production image, we don't need to set `US.UTF-8`, I will delete this lines

**Pull request merged**